bundles/binding/org.openhab.binding.serial/src/main/java/org/openhab/binding/serial/internal/SerialBinding.java

```diff
@@ -135,7 +140,7 @@ public String getBindingType() {
     @Override
     public void validateItemType(Item item, String bindingConfig) throws BindingConfigParseException {
         if (!(item instanceof SwitchItem || item instanceof StringItem || item instanceof NumberItem
-                || item instanceof RollershutterItem)) {
+                || item instanceof RollershutterItem || item instanceof ContactItem || item instanceof DimmerItem)) {
             throw new BindingConfigParseException("item '" + item.getName() + "' is of type '"
```

Comment:
Because of the changes to supported item types, this throw message should probably be updated to something like "Items of type {} are not supported."